src/pages/round/[...number].tsx

```diff
@@ -75,57 +64,80 @@ export default function Round() {
     }
   }, [display, roundList, number]);
 
-  const Url = "/api/v1/bribedata/" + requestedRound;
-
   console.log(roundList.latest);
   console.log(number[0], requestedRound);
-
-  const { data: bribedata, isLoading } = useQuery(["bribedata", Url], () =>
-    fetchRoundData(Url)
-  );
-
-  if (isLoading) {
-    return <Center>Loading ... </Center>;
-  }
+  console.log(bribeData?.header);
+  console.log(bribeData?.bribelist);
```

Comment:
maybe you should check for valid bribeData and display an error message, if not availabe, like
`if (!bribeData) return (<>Error: Data not available, please come back later.</>)` - with some formatting you like. 
Then you do not need the questionmark-operator after each occurance of bribeData(?).
Just a suggestion for your next improvement.